rust-semantics/representation.md

```diff
@@ -23,9 +23,13 @@ module RUST-VALUE-SYNTAX
                     | u128(MInt{128})
                     | tuple(ValueList)
                     | struct(TypePath, Map)  // Map from field name (Identifier) to value ID (Int)
+                    | Range
                     | Bool
                     | String
 
+
+    syntax Range ::= intRange(PtrValue, PtrValue)
```

Comment:
I think that this should be intRange(Value, Value), the pointer part does not help here.